### DISCUSSION TITLE
Add taxtonomy terms to the $pieces array

### DISCUSSION THREAD

**Discussion by BenJackGill** [pods-framework/pods#6683] [Ideas]
## Issue Overview (Feature Suggestion)
The $pieces array passed by the `pods_api_pre_save_pod_item_{podname}` filter does not include information about which taxonomy terms are assigned to the post during save.

It does have an element named [object_fields] which contains details about the _taxonomies_ assigned to the post, but it has no details about which _terms_ are assigned to those taxonomies for the post.

I suggest we add a [taxonomies] element to the $pieces array which contains information about the terms assigned to the post.

## Expected Behavior
Add a [taxonomies] element to the $pieces array which contains information about the terms assigned to the post.

Additionally, allow us to add or remove taxonomy terms on the post by updating the $pieces array.

## Current Behavior
There is no way to check or change the taxonomy terms pre-save.

Currently, we have to use `get_the_terms()` which does not get the terms assigned to the post during save, and therefore will always be pulling old data.

**Comment by codente** [comment]
+1
